Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -375,6 +375,24 @@ extension NSManagedObjectContext: MessageDatabaseSession {
     func delete(message: MessageDTO) {
         delete(message)
     }
+
+    func pin(message: MessageDTO, _ pin: Pin) throws {
```

Comment:
Why is it `_ pin` and not `pin`? 🤷‍♂️ :) 

I think with the adjusted naming it will work nicely `func pin(message: MessageDTO, pinning: MessagePinning)`

Reply:
Yeah the hidden labelling was so it was not so repetitive. `pin(message: someMessage, .noExpirationDate)` instead of repeating `pin` again. But `MessagePinning` sounds good to me 👍